### PR TITLE
Fix admin portal go embed

### PR DESCRIPTION
**Pull request by ellis-johnson**
### Which issue this PR addresses:

Fixes https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/16267498

### What this PR does / why we need it:

When go:embed was introduced to admin portal it broke v2 navigation. This PR addresses that issue and also allows the admin portal e2e tests to run and not throw errors in production

### Test plan for issue:

E2E Tests

### Is there any documentation that needs to be updated for this PR?

No

### CODE REVIEW

**Review by darthhexx — CHANGES_REQUESTED** [review]
Small change to the calculation of `dockerSucceeded`.

File: test/e2e/setup.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ var (
 	clusterName       string
 	clients           *clientSet
 
-	dockerSucceeded bool
+	dockerSucceeded bool = true
```
[the remark this answers is not in the document]
Comment (ellis-johnson) [reply test/e2e/setup.go]: Fixed

File: test/e2e/setup.go [hunk 2]
```diff
@@ -400,7 +400,15 @@ func setup(ctx context.Context) error {
 		return err
 	}
 
-	setupSelenium(ctx)
+	cmd := exec.CommandContext(ctx, "which", "docker")
+	_, err = cmd.CombinedOutput()
+	if err != nil {
+		dockerSucceeded = false
+	}
```
[the remark this answers is not in the document]
Comment (ellis-johnson) [reply test/e2e/setup.go]: Done

File: portal/v2/package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "HTTPS=true react-app-rewired start",
-    "build": "react-app-rewired build",
+    "build": "react-app-rewired build && rm -rf ../../pkg/portal/assets/v2/build && mv -f build ../../pkg/portal/assets/v2/",
```
Comment (cblecker) [inline portal/v2/package.json:22]: As I said above, `rm -rf` isn't a great option. Looks like we can directly override the build path as below.

```suggestion
    "build": "BUILD_PATH=../../pkg/portal/assets/v2/build react-app-rewired build",
```

**Review by cblecker — CHANGES_REQUESTED** [review]

File: portal/v2/package.json (hunk 3 above)
Comment (ellis-johnson) [reply portal/v2/package.json]: Done

File: portal/v1/package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "description": "",
     "private": true,
     "scripts": {
-        "build": "webpack --config webpack.prod.js"
+        "build": "webpack --config webpack.prod.js && rm -rf ../../pkg/portal/assets/v1/build && mv -f build ../../pkg/portal/assets/v1/"
```
[the remark this answers is not in the document]
Comment (ellis-johnson) [reply portal/v1/package.json]: Done

**Review by cblecker — APPROVED** [review]

**Pull request merged**